README.md

```diff
@@ -117,6 +117,8 @@ https://github.com/bytebase/bytebase/tree/main/docs/version-management.md
 
 1. We are maintaining an [online database glossary list](https://bytebase.com/database-glossary/?source=github), you can add/improve content there.
 
+1. Before creating a Pull Request, please follow the [Development Guide](https://github.com/bytebase/bytebase/blob/main/docs/dev-guide.md) for branch and commit message conventions.
```

Comment:
```suggestion
2. Before creating a Pull Request, please follow the [Development Guide](https://github.com/bytebase/bytebase/blob/main/docs/dev-guide.md) for branch and commit message conventions.
```

Reply:
1 is correct per markdown

Reply:
It was meant to be all `1`. IMO, we should respect that.

It leaves the order irrelavent.

Reply:
Got it.

Reply:
It was meant to be all `1`. IMO, we should respect that.

It leaves the order irrelevant.